src/soundio/soundmanager.h

```diff
@@ -107,7 +108,7 @@ class SoundManager : public QObject {
     QList<AudioOutput> registeredOutputs() const;
     QList<AudioInput> registeredInputs() const;
 
-    QSharedPointer<EngineNetworkStream> getNetworkStream() const {
+    const std::unique_ptr<EngineNetworkStream>& getNetworkStream() const {
```

Comment:
Sorry, It seams I was confused about all the pointers when commenting. Please revert this.
getNetworkStream() itself does not know what happens with the pointer so sharing it was somehow correct.